lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -49,32 +85,35 @@ define([
                     results: testTeamData
                 })
             };
-            var teamsView = createTopicTeamsView(options);
-            var footerEl = teamsView.$('.teams-paging-footer');
+            var footerEl;
```

Comment:
Nit: hoist to top? I thought our current linting would yell at you for this.

Reply:
?